### ISSUE TITLE
Error 500 on PR renaming

### ISSUE THREAD

**Issue by BastienVigneron** [go-gitea/gitea#30070]
### Description

When I try to rename a PR (to remove the `WIP:`), I get an HTTP/500 errror.

The log produce the following: 

```
2024/03/25 12:47:32 ...eb/routing/logger.go:102:func1() [I] router: completed GET /iam/iamv2/issues/117/content-history/overview?_csrf=Fvrje1qxFrfNnEsI7pgKjQvSFLg6MTcxMTM3MDc0MzgzMDEyNDkwOA for 10.16.0.64:0, 200 OK in 24.3ms @ repo/issue_content_history.go:24(repo.GetContentHistoryOverview)
2024/03/25 12:47:36 ...rs/web/repo/issue.go:2163:UpdateIssueTitle() [E] ChangeTitle: exit status 128 - fatal: ambiguous argument '': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
 - fatal: ambiguous argument '': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

### Gitea Version

1.21.9

### Can you reproduce the bug on the Gitea demo site?

Yes

### Git Version

The one provided in official docker image.

### Operating System

Official docker image (gitea/gitea)

### How are you running Gitea?

Via official docker image (gitea/gitea). 

### Database

PostgreSQL

**Comment by lunny** [comment]
This has been fixed by #30066 and will be released on v1.21.10

**Comment by wxiaoguang** [comment]
Ref #29997

Sorry for the inconvenience.

* It could be fixed by updating to pre-1.21.10:
  * Docker image: use gitea/gitea:1.21-nightly and gitea/gitea:1.21-nightly-rootless
  * Binary: download from https://dl.gitea.com/gitea/1.21/
